<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteWidget</class>
 <widget class="QWidget" name="RouteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>454</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RouteWidget {
	background-color: #f2f2f2;
}

QWidget[objectName^=&quot;roundedWidget&quot;] {
	background-color: #ffffff; 
	border-radius: 24px;
}

QPushButton {
    background-color: #010E59;
	border: none;
	border-radius: 24px;
    color: #ffffff;
    font-family: &quot;Inter Medium&quot;, sans-serif;
    font-size: 18px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="roundedWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>210</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="leftMargin">
           <number>36</number>
          </property>
          <property name="topMargin">
           <number>36</number>
          </property>
          <property name="rightMargin">
           <number>64</number>
          </property>
          <property name="bottomMargin">
           <number>36</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="departureTime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #010E59;</string>
              </property>
              <property name="text">
               <string>04:15</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="departureDate">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #6D6D6D; </string>
              </property>
              <property name="text">
               <string>15 окт, ср</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="departurePlace">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Полоцк</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_33">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="label_40">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../resources.qrc">:/icons/icons/circle.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_13">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_41">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../resources.qrc">:/icons/icons/filled-circle.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="travelTime">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #6D6D6D; </string>
              </property>
              <property name="text">
               <string>5 ч. 35 мин в пути</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="arivingTime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #010E59;</string>
              </property>
              <property name="text">
               <string>09:50</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="destinationDate">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #6D6D6D; </string>
              </property>
              <property name="text">
               <string>15 окт, ср</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="destinationPlace">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Минск</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_12">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="leftMargin">
           <number>18</number>
          </property>
          <property name="rightMargin">
           <number>72</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>18</number>
            </property>
            <item>
             <widget class="QLabel" name="routePrice_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #010E59;</string>
              </property>
              <property name="text">
               <string>25.00 руб</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="placeCount_label">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #6D6D6D;</string>
              </property>
              <property name="text">
               <string>15 мест осталось</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="buyRoute_btn">
            <property name="minimumSize">
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Купить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="detail_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1600</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f2f2f2;
border-bottom-left-radius: 24px;
border-bottom-right-radius: 24px;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>36</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="details_btn">
           <property name="font">
            <font>
             <family>Inter-Regular</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #010E59;
font-family: Inter-Regular;
font-size: 12;</string>
           </property>
           <property name="text">
            <string>Детали рейса</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="routeDetail_arrow">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources.qrc">:/icons/icons/arrow-down.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>974</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="roundedWidget_routeDetail" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top-left-radius: 0;
border-top-right-radius: 0;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="leftMargin">
       <number>36</number>
      </property>
      <property name="topMargin">
       <number>36</number>
      </property>
      <property name="rightMargin">
       <number>72</number>
      </property>
      <property name="bottomMargin">
       <number>36</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources.qrc">:/icons/icons/geo2.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: transparent;
color: #010E59;
</string>
          </property>
          <property name="text">
           <string>Маршрут</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Рейс № </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="routeNumber">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>338</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string> по маршруту </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pathNumber">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>327б</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="route">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Полоцк - Минск</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>На </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="routeDate">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>15.02.2026</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string> в </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="routeTime">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>15.02.2026</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="stopsLayout">
        <property name="topMargin">
         <number>16</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>details_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>roundedWidget_routeDetail</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
